Sources/App/configure.swift

```diff
@@ -64,15 +64,22 @@ public func configure(_ app: Application) async throws {
     // cors middleware should come before default error middleware using `at: .beginning`
     app.middleware.use(cors, at: .beginning)
 
-    app.databases.use(DatabaseConfigurationFactory.postgres(configuration: .init(
-        hostname: Environment.get("DATABASE_HOST") ?? "localhost",
-        port: Environment.get("DATABASE_PORT").flatMap(Int.init(_:)) ?? (app.environment == .testing ? 5433 : 5432),
-        username: Environment.get("DATABASE_USERNAME") ?? "vapor_username",
-        password: Environment.get("DATABASE_PASSWORD") ?? "vapor_password",
-        database: Environment.get("DATABASE_NAME") ?? "vapor_database",
-        tlsConfiguration: app.environment == .testing ? .none : .forClient(certificateVerification: .none))
-    ), as: .psql)
-
+	var tlsConfig: TLSConfiguration = .makeClientConfiguration()
+	// Do not disable certificate verification in production. You should provide a certificate to the TLSConfiguration to verify against
+	tlsConfig.certificateVerification = .none
```

Comment:
Thanks! Maybe we should keep the certificate verification on by default at least for release builds? Any reason why not to?

Reply:
Just made a new commit, this should be a [better solution](https://github.com/petrpavlik/vapor-saas-backend-template/pull/5/commits/6b1d5854c7984c5e64ec6f4a20f3c840ab6fd659)